include/utransport/UAttributes.h

```diff
@@ -0,0 +1,299 @@
+                        {
+                            return token_;
+                        }
+
+                        return std::nullopt;
+                    }
+
+                    /**
+                    * How long this event should live for after it was generated (in milliseconds).
+                    * Events without this attribute (or value is 0) MUST NOT timeout.
+                    * @return Returns an Optional time to live attribute.
+                    */
+                    std::optional<USerializationHint> serializationHint() const {
+
+                        if (hint_)
+                        {
+                            return hint_;
+                        }
+                        
+                        return std::nullopt;
+                    }
+
+                    /**
+                    * an explicit destination URI.
+                    * @return Returns an Optional destination URI attribute.
+                    */
+                    std::optional<UUri> sink() const {
+
+                        if (sink_)
+                        {
+                            return *sink_;
+                        }
+                        
+                        return std::nullopt;
+                    }
+
+                    /**
+                    * The reqid is used to return a response for a specific request.
+                    * @return Returns an Optional requestId attribute.
+                    */
+                    std::optional<UUID> reqid() const {
+
+                        if (!reqid_)
+                        {
+                            return reqid_;
+                        }
+                        
+                        return std::nullopt;
+                    }
+
+                    /**
+                    * The permission level of the message.
+                    * @return Returns an Optional permission level attribute.
+                    */
+                    std::optional<int32_t> plevel() const {
+
+                        if (plevel_)
+                        {
+                            return plevel_;
+                        }
+                        
+                        return std::nullopt;
+                    }
+
+                    /**
+                    * The communication status of the message.
+                    * @return Returns an Optional communication status attribute.
+                    */
+                    std::optional<int32_t> commstatus() const {
+
+                        if (commstatus_)
+                        {
+                            return commstatus_;
+                        }
+                        
+                        return std::nullopt;
+                    }
+
+                    class UAttributesBuilder
+                    {
+                        public:
+                            UAttributesBuilder(UUID id, UMessageType type, UPriority priority) {
+
```

Comment:
sometime use 
``
this->privatefile_
``
sometime you use 
``
privatefield_
``
Can we consistent?